js/search/search.js

```diff
@@ -71,8 +71,8 @@ class Search {
     init() {
         libSymphonySearch.symSEInit();
         libSymphonySearch.symSEEnsureFolderExists(this.dataFolder);
-        libSymphonySearch.symSERemoveFolder(this.realTimeIndex);
-        libSymphonySearch.symSERemoveFolder(this.batchIndex);
+        Search.deleteFolderRecursive(this.realTimeIndex);
+        Search.deleteFolderRecursive(this.batchIndex);
```

Comment:
You can probably name this as "deleteIndexFolders", which is easier to understand.